js_modules/dagster-ui/packages/ui-core/src/instigation/TickHistory.tsx

```diff
@@ -55,24 +58,22 @@ Chart.register(zoomPlugin);
 type InstigationTick = HistoryTickFragment;
 
 const PAGE_SIZE = 25;
-interface ShownStatusState {
-  [InstigationTickStatus.SUCCESS]: boolean;
-  [InstigationTickStatus.FAILURE]: boolean;
-  [InstigationTickStatus.STARTED]: boolean;
-  [InstigationTickStatus.SKIPPED]: boolean;
+
+enum TickStatusDisplay {
+  ALL = 'ALL',
+  FAILED = 'FAILED',
+  SUCCESS = 'SUCCESS',
```

Comment:
you could do just 
```
```suggestion
  ALL,
  FAILED,
  SUCCESS,
```